pittgoogle/schema.py

```diff
@@ -604,6 +604,9 @@ def _name_in_bucket(alert: "Alert"):
         import astropy.time  # always lazy-load astropy
 
         _date = astropy.time.Time(alert.get("mjd"), format="mjd").datetime.strftime("%Y-%m-%d")
+        # each diaSource will be associated with either a diaObject or ssObject
+        if not alert.objectid:
+            return f"{alert.schema.version}/{_date}/{alert.attributes['ssSource_ssObjectId']}/{alert.sourceid}.avro"
         return f"{alert.schema.version}/{_date}/{alert.objectid}/{alert.sourceid}.avro"
```

Comment:
With the changes we made to `Alert.objectid` today plus the decision to including the key names here, I think we now want the following. But double check me 😃 .

```suggestion
        objectid_key = alert.get_key("objectid", name_only=True)
        sourceid_key = alert.get_key("sourceid", name_only=True)
        return f"{alert.schema.version}/{_date}/{objectid_key}={alert.objectid}/{sourceid_key}={alert.sourceid}.avro"
```

Reply:
Yes, this is what we agreed on! Confirming that it works as expected.